nodejs/envconfig.json

```diff
@@ -0,0 +1,24 @@
+[{
+    "version": "1.0.0", 
+    "runtimeVersion" : "1.0.0", 
```

Comment:
Runtime version is definitely 1.0.0 - Can you please check?

Reply:
Changed